### PR TITLE
Added: Add admin interface app for global admin dashboard theming and styling

### LINKED ISSUE #894: Themed admin interface
Could we add AML-like theming to the admin interface? It is not very high priority, but it would be nice.

### PR DESCRIPTION
**Pull request by drikusroor**
This pull request adds a new "admin_interface" app that allows for theming and styling of the global admin dashboard. 

For now, I've only added the css color variables that are used by Django and the custom css stylesheet. If we also want to set fonts, I can add it in a later PR.

Architecturally, the theme (`AdminInterfaceThemeConfiguration`) hangs under the admin interface configuration (`AdminInterfaceConfiguration`) with a one-to-one relation. I decided to do it like this so we could potentially add other kinds of configuration features to this admin interface configuration model if needed in the future. The theming part (colors, and possibly fonts in the future) will then be nicely separated from potential unrelated settings.

Resolves #894

### Overview

image

### Edit form

image

### CODE REVIEW

**Review by BeritJanssen — COMMENTED** [review]
I love that you managed to implement a fully customizable admin interface, and kudos to all the effort that went into this but ... I do think 500 lines of code added for functionality that may not really get used (i.e., who will really care enough about how the admin interface looks to set 30+ color variables?) is a bit overkill. It pains me, but I'd vote for a css file to style everything "AML style" and leave this branch for what it is.

File: backend/admin_interface/static/admin_interface/js/admin_interface_configuration.js
Comment (BeritJanssen) [inline backend/admin_interface/static/admin_interface/js/admin_interface_configuration.js:1]: Why is everything commented out here?

Comment (drikusroor) [reply backend/admin_interface/static/admin_interface/js/admin_interface_configuration.js:1]: It's an unused piece of code and I'm removing it.

**Comment by drikusroor** [comment]
Should I merge this?

**Pull request merged**